client/entrypoints/old-settings-upe-toggle/index.js

```diff
@@ -0,0 +1,37 @@
+/* global wc_stripe_old_settings_param */
+import { dispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import domReady from '@wordpress/dom-ready';
+
+domReady( () => {
```

Comment:
Since we need to display the snackbar _after_ the settings are saved, I am enqueuing this script for the short time we'll display the old settings.

Maybe we can use the same script for https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1888 if we can't have tracking on the server side? (we would just need to add the tracking here, all the rest is the same. The script is already enqueued if the UPE flag is toggled, not just if it's disabled)

Reply:
Looks good as an interim solution. :100: 

I was struggling to do the tracking in the PHP way. Like the idea of implementing the tracking here. I will update my one on top of this branch. Thanks for the insight :+1: